core/src/pollers/poll_buffer.rs

```diff
@@ -127,7 +115,6 @@ where
     /// Returns `None` if the poll buffer has been shut down
     #[instrument(name = "long_poll", level = "trace", skip(self))]
     async fn poll(&self) -> Option<pollers::Result<T>> {
-        self.polls_requested.add_permits(1);
         if let Some(fun) = self.num_pollers_changed.as_ref() {
```

Comment:
Should this metric reflect the number of active polls? Why only emit it when lang polls?

Reply:
Yup, this needs to change now

Reply:
This is changed